server/server_test.go

```diff
@@ -159,7 +159,7 @@ func TestOAuth2CodeFlow(t *testing.T) {
 	clientSecret := "testclientsecret"
 	requestedScopes := []string{oidc.ScopeOpenID, "email", "profile", "groups", "offline_access"}
 
-	t0 := time.Now()
+	t0 := time.Now().UTC()
```

Comment:
Discussed offline. Removing the UTC to increase test coverage to include cases when time zones are involved.